docs/core/deploying/fastening-libraries.md

```diff
@@ -12,7 +12,7 @@ ms.devlang: dotnet
 ms.assetid: 4424a947-bdf9-4775-8d48-dc350a4e0aee
 ---
 
-# How to manage package dependency versions for .NET Core 1.0
+# How to Manage Package Dependency Versions for .NET Core 1.0
```

Comment:
want to change the filename to match the title a bit better? "fastening" isn't a typical term.